src/storage/mvcc/reader/reader.rs

```diff
@@ -212,10 +212,14 @@ impl<S: Snapshot> MvccReader<S> {
         Ok(())
     }
 
+    /// Gets the value of the specified key's latest version before specified `ts`.
+    /// It tries to ensure the write record's `gc_fence`'s ts, if any, greater than specified
+    /// `gc_fence_limit`. Pass 0 to `gc_fence_limit` to skip the check.
```

Comment:
Should it be None instead of 0?

Reply:
Oh sorry I forgot to update it while I'm changing my code